tests/emitters/evolution_strategy_emitter_test.py

```diff
@@ -35,7 +35,20 @@ def test_dtypes(dtype):
 
     # Try running with the negative sphere function for a few iterations.
     for _ in range(10):
-        sols = emitter.ask()
-        objs = -np.sum(np.square(sols), axis=1)
-        bcs = sols[:, :2]
-        emitter.tell(sols, objs, bcs)
+        solution_batch = emitter.ask()
+        objective_batch = -np.sum(np.square(solution_batch), axis=1)
+        measures_batch = solution_batch[:, :2]
+
+        # Add solutions to the archive.
+        status_batch = []
```

Comment:
we'll have to remember to shorten this once we get add_batch -- can you create a TODO?